base/multidimensional.jl

```diff
@@ -863,3 +863,193 @@ If `dim` is specified, returns unique regions of the array `itr` along `dim`.
         @nref $N A d->d == dim ? sort!(uniquerows) : (1:size(A, d))
     end
 end
+
+"""
+    C, ia, ib, ic = uniqueind(itr, dim)
+
+A function that operates similiarly to `unique(itr,dim)` but returns multiple
+output arguments having the following properties:
+
+C - the unique elements of the array `itr` along the selected dimension `dim`
+ia - a Vector{Int} of indices such that:
```

Comment:
> Maybe we can find a short way of computing it from ic, which could be given in the docs for those who need it?

For people coming to Julia from K/q, having to execute two functions to get back the results of what is a single character operator (`=A`) applied to an array is incredibly verbose.  For that part of the audience, a single function call will likely be preferred.